### PR TITLE
Feature/edit show bootstrapped

### PR DESCRIPTION
**Pull request by p0psicles**
** Branched off #4075 

- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

Purpose of this PR is to "fix" the bootstrapping (css) of the config elements, like fileBrowser, inputs, checkboxes.

I started with editShow, as that also uses some new vue components like select-list. But the changes effect the entire UI. So this will be extended to also all config pages.

## TODO:
- [ ] ~Make config responsive~
- [x] Use bootstrap form-groups with glyphs
- [x] Remove .btn class overrides from styles (this breaks allot of bootstrap combined elements)
- [x] Remove hard width, use max-width in stead.

### CODE REVIEW

File: themes-default/slim/views/vue-components/anidb-release-group-ui.mako [hunk 1]
```diff
@@ -48,7 +54,7 @@
                 <span>Release groups</span>
                 <ul>
                     <li v-for="release in itemsReleaseGroups" class="initial" v-bind:class="{active: release.toggled}" @click="release.toggled = !release.toggled">{{ release.name }}</li>
-                    <div class="arrow" @click="moveToList('releasegroups')">
+                    <div v-if="newGroup === ''" class="arrow" @click="moveToList('releasegroups')">
```
Comment (OmgImAlexis) [inline themes-default/slim/views/vue-components/anidb-release-group-ui.mako]: Use `!newGroup`.

File: themes-default/slim/views/vue-components/file-browser.mako [hunk 2]
```diff
@@ -262,11 +275,22 @@ Vue.component('file-browser', {
             return resultField;
         };
 
-        $(this.$el).children('input').fileBrowser({
+        $(this.$refs.locationInput).fileBrowser({
             title: this.title,
-            localStorageKey: 'postprocessPath'
+            localStorageKey: this.localStorageKey
         });
```
Comment (OmgImAlexis) [inline themes-default/slim/views/vue-components/file-browser.mako]: Change to this.

```
const { title, localStorageKey } = this;
$(this.$refs.locationInput).fileBrowser({
    title,
    localStorageKey
});
```

**Comment by p0psicles** [comment]
I'll limit the scope of the PR to editShow.
I'll change the rest of config in a new PR.

**Comment by sharkykh** [comment]
@p0psicles Looks neat! Added it to my main install for testing.
1. The font size in the input fields seems a little too big - it is cut off on the bottom, see the `g` in `English`
2. When I click on the  and then go back to the normal list it adds an empty item:
image

**Comment by p0psicles** [comment]
Yeah i noticed the font size. Ill look into that. Ill fix the select-list issues

**Comment by p0psicles** [comment]
I think that for the font size we should make a decision for the entire ui. For now the other issues should be resolved.

**Comment by sharkykh** [comment]
> 3. If I write values and then click , it doesn't make them comma separated?

The above is still true for current develop, but it's not that important right now

**Comment by p0psicles** [comment]
Sorry, then I don't understand. Can you be a little bit more specific in reproducing this?

**Comment by sharkykh** [comment]
Sure.

**Comment by p0psicles** [comment]
Yeah, that's by design, you have to add the value, using the plus.

**Comment by sharkykh** [comment]
You mean you have to save it first? That means you currently have to refresh the page in order to get the value as comma separated values. Just saying.

**Pull request merged**